### DISCUSSION TITLE
Saving parameters as a template in MoBi

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1232] [Questions & Problems] [answered]
Hi all,

I want to add a set of parameters to each organ in the human spatial structure. Is there a way, after creating this for one organ using a relative path, to save this set of parameters as a template and then import it to each other organ? 

I see there is a button of 'load parameter' option, but I can't figure out how to save the parameters to a template.

Kind regards
Salma

**Comment by StephanSchaller** [comment, marked as answer]
Hi @SalmaBahnasawy. One option would be to save the organ or compartment as PKML and then load the parameter in the other organs from this saved PKML file (you can multiselect):

1. 
![image](https://user-images.githubusercontent.com/25477643/174328984-13ecdd9b-36aa-45a8-9715-38309fa9b4ea.png)

2.  
![image](https://user-images.githubusercontent.com/25477643/174329133-46f655cb-d2b6-450a-990b-79b7a7006e28.png)

or

![image](https://user-images.githubusercontent.com/25477643/174329216-46aeba46-cebe-44c7-b71c-4534ac5f1ca6.png)

3. 
![image](https://user-images.githubusercontent.com/25477643/174329384-4e12af11-73cb-4a84-a30b-18a6813f171e.png)

**Reply by bahsa505** [reply]
Thanks a lot, Stephan. It worked quite well!

**Comment by Fate-afshar** [comment]


**Reply by msevestre** [reply]
Can you move this answer to the right discussion? Thanks
